src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs

```diff
@@ -8,16 +8,18 @@
 using Microsoft.DotNet.Maestro.Client.Models;
 using Microsoft.Extensions.Logging;
 using System;
-using System.Net;
 using System.Threading.Tasks;
-using Microsoft.DotNet.Maestro.Client;
 using System.Linq;
 using System.Collections.Generic;
 
 namespace Microsoft.DotNet.Darc.Operations
 {
     internal class AddBuildToChannelOperation : Operation
     {
+        private int BuildPromotionPipelineId { get; } = 750;
```

Comment:
I think this can be made a const.

Reply:
And the ones below.

Reply:
Looks like I can't use `const` for `properties` so I decided to use "get-only" properties. I'll make them a `const field`.